tonic/src/lib.rs

```diff
@@ -122,3 +122,5 @@ pub(crate) type Error = Box<dyn std::error::Error + Send + Sync>;
 #[cfg(feature = "codegen")]
 #[cfg_attr(docsrs, doc(cfg(feature = "codegen")))]
 pub mod codegen;
+
+pub type Result<T> = std::result::Result<T, Status>;
```

Comment:
This way users can still override the error type if they want.

```suggestion
pub type Result<T, E = Status> = std::result::Result<T, E>;
```

Reply:
Also missing docs.

Reply:
TIL